applications/crossbar/src/modules/provisioner_util.erl

```diff
@@ -201,9 +201,15 @@ maybe_delete_account(Context) ->
 maybe_send_contact_list(Context) ->
     maybe_send_contact_list(Context, cb_context:resp_status(Context)).
 maybe_send_contact_list(Context, 'success') ->
-    _ = case get_provisioning_type() of
+    _ = case cb_context:is_context(Context)
```

Comment:
Why the need for this check? This function seems to be only called from cb_accounts with a Context record.